app/views/merchant_items/show.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1>Merchant Item Details</h1>
+
+<h3>Merchant: <%= @merchant.name %></h3>
+
+<p><b>Item Name: </b><%= @item.name %></p>
+<p><b>Item Description: </b><%= @item.description %></p>
+<p><b>Current Selling Price: </b><%= number_to_currency(@item.unit_price / 100.to_f) %></p>
```

Comment:
So if the controller went on a diet you could replace the "@item"s with "@merchant.items.find(params[:item_id])"s and everything would work fine.